.travis.yml

```diff
@@ -14,7 +14,7 @@ env:
   global:
     - MOVEIT_CI_TRAVIS_TIMEOUT=85  # Travis grants us 90 min, but we add a safety margin of 5 min
     - ROS_DISTRO=noetic
-    - ROS_REPO=ros
+    - ROS_REPO=ros-testing
```

Comment:
Oh, I didn't realize that was what this variable was for.  That is nice, thank you.